src/stdpar/plugin-PixelTriplets/gpuPixelDoublets.h

```diff
@@ -99,30 +99,211 @@ namespace gpuPixelDoublets {
                                 CellTracksVector* cellTracks,
                                 TrackingRecHit2DSOAView const* __restrict__ hhp,
                                 GPUCACell::OuterHitOfCell* isOuterHitOfCell,
-                                int nActualPairs,
+                                int nPairs,
                                 bool ideal_cond,
                                 bool doClusterCut,
                                 bool doZ0Cut,
                                 bool doPtCut,
                                 uint32_t maxNumOfDoublets) {
     auto const& __restrict__ hh = *hhp;
-    doubletsFromHisto(layerPairs,
-                      nActualPairs,
-                      cells,
-                      nCells,
-                      cellNeighbors,
-                      cellTracks,
-                      hh,
-                      isOuterHitOfCell,
-                      phicuts,
-                      minz,
-                      maxz,
-                      maxr,
-                      ideal_cond,
-                      doClusterCut,
-                      doZ0Cut,
-                      doPtCut,
-                      maxNumOfDoublets);
+    // ysize cuts (z in the barrel)  times 8
```

Comment:
(note to self) This is a direct copy from `gpuPixelDoubletsAlgos.h`.

Can that header be removed now?